test/runtests.jl

```diff
@@ -1,315 +1,556 @@
 using Base.Test, ODBC, DataStreams, Missings, WeakRefStrings, DataFrames, DecFP
 
+# To run these tests first run these statements from the command line in the tests directory:
+# odbcinst -i -s -h -f mytest.odbc.ini
+# odbcinst -i -s -h -f pgtest.odbc.ini
+# odbcinst -i -s -h -f ./test/mssqltest.odbc.ini
+
+# You can modify the above files as needed to point to your mysql, postgres, or mssql
+# database and drivers, or to specify your username and password
+# Note:
+# Microsoft drivers don't support reading usernames and passwords from .odbc.ini files so
+# you have to specify those in the DSN setup call:
+# ODBC.DSN("MSSQL-test", "yourusername", "yourpassword")
+
+TEST_MYSQL = true
+TEST_POSTGRESQL = true
+TEST_MSSQL = true
```

Comment:
Consider making these into a single ENV variable so it can be controlled without having to modify the code. Maybe:
```julia
const TEST_DRIVERS = split(get(ENV, "ODBC_TEST_DRIVERS", "mysql, postgresql , mssql"), r"\s*,\s*")
const TEST_MYSQL = "mysql" in TEST_DRIVERS
...
```